aspnetBlazor/docker-compose.yml

```diff
@@ -16,3 +16,91 @@ services:
     image: ${DOCKER_LOGIN}/__container__:${TAG}
     ports:
       - 7001:7001
+
+  __container__-browser-debug:
+    build:
+        context: .
+        dockerfile: Dockerfile.debug
+    image: ${LOCAL_REGISTRY}:5002/__container__-debug:${TAG}
+    ports:
+      - 2222:2222
+      - 7001:7001
+    depends_on:
+      - cog
+
+  blazor-browser:
+    build:
+        context: .
+        dockerfile: Dockerfile.debug
+    image: ${LOCAL_REGISTRY}:5002/__container__:${TAG}
+    ports:
+      - 7001:7001
+    depends_on:
+      - cog
+
+  cog:
+    image: commontorizon/cog${GPU}:4-rc
```

Comment:
Can we use the `torizon/cog` one? Seems that for rc there is one [here](https://hub.docker.com/r/torizon/cog-imx8/tags)

Reply:
nope, because the `torizon/cog` does not have the wait for the service script, check https://github.com/commontorizon/Containerfiles/tree/main/cog